clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -49,6 +50,7 @@ class DeployKubernetesAtomicOperationDescription extends KubernetesAtomicOperati
   Integer sequence
   KubernetesPodSpecDescription podSpec
   List<KubernetesPersistentVolumeClaimDescription> volumeClaims
+  Map<String, String> customTemplateMetadataLabels
```

Comment:
Which labels do these correspond to?

Reply:
This label for spec.template.metdata.lables which will use for spec.selector.matchLabels.  I don't believe spinnaker support custom label yet.  I can remove it if you feel that it might be some potential issues.

Reply:
I would remove them, we have avoided custom labels for this provider version because of all the potential issues.